src/Content/Text/BBCode.php

```diff
@@ -1064,6 +1067,44 @@ private static function removePictureLinksCallback(array $match): string
 		return $text;
 	}
 
+	/**
+	 * Remove invalid parts from an URL
+	 *
+	 * @param string $url
+	 * @return string sanitized URL
+	 */
+	private static function sanitizedLink(string $url): string
```

Comment:
```suggestion
	private static function sanitizeLink(string $url): string
```

Reply:
I changed that and moved the functionality, since it is now used at several places.